components/definition/cellRules/NetPyNECellRules.js

```diff
@@ -286,6 +320,29 @@ export default class NetPyNECellRules extends React.Component {
     }
   }
 
+  handleHierarchyClick = (nextPage) => {
+    const { page, selectedCellRule, value } = this.state;
```

Comment:
I am bit scared of this method. Can we simplify it somehow? Maybe change the swithc with a if/else and double check if we can simplify some conditions or merge two of them... Some comments will be helpful when we look at this code in a few weeks as well